error-handlers/error-handler.js

```diff
@@ -1,3 +1,9 @@
 exports.error_handler = (req, res) => {
   res.status(404).send({ msg: "path is not correct" });
 };
+
+exports.handle_custom_errors = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "bad request" });
+  }
+};
```

Comment:
This error code is coming from a PSQL error, so should really be in a PSQL error handler rather than a custom error handler. Save your custom handler for your own Promise.rejects()